core/modules/file/file.module

```diff
@@ -429,6 +429,15 @@ function file_menu() {
     'file' => 'file.pages.inc',
   );
 
+  $items['admin/reports/status/classify'] = array(
+    'title' => 'Classify file types',
+    'page callback' => 'backdrop_get_form',
+    'page arguments' => array('file_type_classify_confirm'),
+    'access arguments' => array('access administration pages'),
+    'type' => MENU_CALLBACK,
```

Comment:
Also from the UTF8-MB4 update:

```suggestion
    'type' => MENU_VISIBLE_IN_BREADCRUMB,
```

This keeps the breadcrumb working unlike `MENU_CALLBACK` but does not show up in menus.

Reply:
Done